grocerylist.py

```diff
@@ -68,20 +70,35 @@ def user_lst():
             food_lst.append(item)
     return food_lst
 
+def save_lst():
+#If no existing list, then save list
+#If existing list, prompt user to save as new list or replace existing list?
 
-# save = input('Would you like to save this list? Y/N: ')
-# if save == 'Y':
-#   fp.writelines(food_lst)
+#Prompt user to save list
+    save = input('Would you like to save this list? Y/N: ')
+#If user selects to save new list
+    if save == 'Y':
+        fp = open(r'C:\Users\Pookie\Downloads\python\grocery_list.txt', 'w')
+#Save each item in list on a new line in .txt
+        for item in food_lst:
+            fp.write("%s\n" % item)
+        print(f"List Saved")
```

Comment:
could you explain why you are using `print(f"List Saved")` instead of `print("list Saved")`? mostly curious!